live-examples/js-examples/array/array-concat.html

```diff
@@ -2,7 +2,11 @@
 <code id="static-js">var array1 = ['a', 'b', 'c'];
 var array2 = ['d', 'e', 'f'];
 
-console.log(array1.concat(array2));
+console.log("First Log Outpur: ",array1.concat(array2));
 // expected output: Array ["a", "b", "c", "d", "e", "f"]
+
+// `...` operator helps to do the same concat, only the syntax is different.
+var array3 = [...array1,'D', 'E', 'F', 'G'];
```

Comment:
This would be better suited to an example related to the spread operator. This example is specific to `Array.concat`. I would, therefore, remove lines 8 - 10